pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -104,7 +101,6 @@ func NewController(client client.Client, record record.EventRecorder) (*Controll
 		client: client,
 		record: record,
 		helper: helper,
-		clock:  realClock,
```

Comment:
And this?

Reply:
Should i add it back ?

Reply:
I don't know. If it's not need just explain :)

Reply:
But I think this might be the reason why tests aren't working.